src/endpoints/Newsrooms/types.ts

```diff
@@ -37,11 +37,11 @@ export interface CreateRequest {
     /**
      * Uploadcare Image JSON
      */
-    newsroom_logo?: string;
+    newsroom_logo?: string | null;
     /**
      * Uploadcare Image JSON
      */
-    square_logo?: string;
+    square_logo?: string | null;
```

Comment:
I don't think the `null` is needed in the `CreateRequest` because if you don't want to provide a logo, you can just omit the property completely.

Reply:
https://github.com/prezly/javascript-sdk/pull/313/commits/0e36cc48f816140596b862cf8a9489cb46bc6d43